controllers/configurationpolicy_controller_test.go

```diff
@@ -246,6 +248,28 @@ func TestConvertPolicyStatusToString(t *testing.T) {
 	assert.NotNil(t, policyInString)
 }
 
+func TestConvertPolicyStatusToStringLongMsg(t *testing.T) {
+	msg := "Do. Or do not. There is no try."
+	for len([]rune(msg)) < 1024 {
+		msg += " Do. Or do not. There is no try."
+	}
+
+	samplePolicy := getSamplePolicy()
+
+	samplePolicy.Status = policyv1.ConfigurationPolicyStatus{
+		ComplianceState: "Compliant",
+		CompliancyDetails: []policyv1.TemplateStatus{
+			{
+				ComplianceState: policyv1.NonCompliant,
+				Conditions:      []policyv1.Condition{{Message: msg}},
+			},
+		},
+	}
+	statusMsg := convertPolicyStatusToString(&samplePolicy)
+
+	assert.Contains(t, statusMsg, "...")
```

Comment:
Is it possible to also assert its length?

Reply:
Good point!